jaguar-gpu/Makefile

```diff
@@ -1,9 +1,9 @@
 NAME=jaguar-gpu
-R2_PLUGIN_PATH=$(shell r2 -hh|grep R2_LIBR_PLUGINS|awk '{print $$2}')
```

Comment:
I recommend to just remove the Makefile and use Meson build, like keystone plugin: https://github.com/rizinorg/rizin-extras/blob/master/keystone/meson.build

Reply:
I've been trying to write one, but wasn't able to get it working completely yet. Can you take a look and see what I'm missing?

Reply:
I have added the meson.build, which works, but I had put the plugin path manually, which is a bit clumsy. Let me know if that needs to/can be changed.

Reply:
I think @ret2libc is more qualified for this question.

Reply:
Yeah this is something we need to figure out... I think probably all plugins should have a meson_options.txt which allows to specify the plugins_directory, but for now I think you can use `~/.local/share/rizin/plugins`.

I'll try to think about these problems soon as i switch back to rz-pm.